doc/users/install.rst

```diff
@@ -14,7 +14,7 @@ You can follow the `Nipype tutorial <https://miykael.github.io/nipype_tutorial/>
 or use this docker container: `docker pull nipype/nipype`
 
 or if you want to build custom docker containers with specific versions of
-software see `Neurodocker <https://github.com/kaczmarj/neurodocker>`_
+software see Neurodocker_. Read the `brief tutorial <neurodocker.html>`_.
```

Comment:
@kaczmarj - i don't think the html is necessary.

just: `brief tutorial <neurodocker_tutorial>`_.

Reply:
@satra - i could not get that to render properly, so i used :doc:`neurodocker` instead.